dotCMS/src/main/java/com/dotcms/security/apps/AppsAPI.java

```diff
@@ -16,114 +16,116 @@
 import java.util.Set;
 
 /**
- * Entry Point to manage secrets and third party service integrations
+ * Entry Point to manage secrets and third party App integrations
  */
-public interface ServiceIntegrationAPI {
+public interface AppsAPI {
+
+     static final String APPS_API_IMPL = "APPS_API_IMPL";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary modifiers 'static final' on field 'APPS_API_IMPL': the field is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5218098)